common/src/main/java/io/druid/metadata/MetadataStorageConnectorConfig.java

```diff
@@ -41,6 +41,25 @@
   @JsonProperty("password")
   private PasswordProvider passwordProvider;
 
+  public MetadataStorageConnectorConfig(){
```

Comment:
this class can probably removed, since we decided to make this a simple indexing task